scope3_methodology/ad_tech_platform/yaml_keys.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+""" ATP Model Yaml Keys """
+from dataclasses import dataclass
+
+
+@dataclass
+class AdTechPlatformKeys:
```

Comment:
you are also allowed to just switch the yaml format to be snake case if that is easier & the team's preference!

Reply:
Yeah, I'll switch that now. Was also going to put up a follow up PR that rearranges the yaml slightly as in https://www.notion.so/scope3/Public-Methodology-Setup-0ee430303d9144e99f942f7f019d3404#3af1bf647df448d0b4095c0a9e40e8f9